src/SnapshotTestCase.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Spatie\Snapshots;
+
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Class SnapshotTestCase.
+ *
+ * Special test case, supported Windows OS.
+ *
+ * @see {https://docs.microsoft.com/en-us/windows/desktop/fileio/naming-a-file}
+ */
+abstract class SnapshotTestCase extends TestCase
```

Comment:
There's no need to introduce this abstract class. Move everything to the existing trait.